### DISCUSSION TITLE
Feature Request for Frigate .14 - Mute or Unmute Config Persistence

### DISCUSSION THREAD

**Discussion by moogoos** [blakeblackshear/frigate#12715] [Ideas]
For live view or playback, cameras with audio enabled always default to muted. Once it's selected it stays for the duration of the view. Once you leave the view it's reset back to muted. It would be nice if this was a toggle option or a config option. In my current DVR it's always on so I can jump around freely without clicking the mute or unmute over and over while I cycle through live views or recordings across different cameras

**Comment by hawkeye217** [comment]
It already works correctly in the History view in 0.14, but I've put up a PR for mute/unmute persistence (per browser session only) in the Live view. This won't make it into the initial release of 0.14, but will be available in a future version of Frigate.

https://github.com/blakeblackshear/frigate/pull/12727

**Comment by d-steve** [comment]
Mute persistence seems to work well for live view in 0.14.1.  However, it's not working for me for review playback or history.

Mac OS 14.6.1, Safari 17.6, Chrome 128.0.6613.86:
Live view:
- Start new session. Go to a camera live view, unmute. Leave live view (ex: go to frigate+), return => still unmuted. nice.

History view:
- Still in camera live view, now click on History, unmute. Click Back => Live view still unmuted. nice. Click History  => automatically muted again. :(
- Now unmute and switch to another camera while still in history view  => still unmuted. nice. 

Review view:
- Switch to Review. Select a review item, unmute. switch cameras within this view => still unmuted. nice.
- Unmute this review item, switch to another camera while still in the review item => still unmuted. nice.
- Back to Review, select same or different review item => automatically muted again. :(

Seems that Live view is browser session-based, while history and review views are not. Is there a browser setting this is tied to, or am I missing somethinge else? Thx

**Reply by hawkeye217** [reply]
To be clear, the "Review view" brings you to History view when clicking on a review item. So they are one in the same. When you enter History view and stay in History view, your muting/unmuting is persisted. When you leave History view, it resets.

My PR that I linked above only made changes to Live view. I'm not sure we want to make changes to History view, but maybe the other devs have a different opinion.

**Reply by moogoos** [reply]
would it be possible to make this an option in the config and just default to muted for both history and live view if an override isn't made in the config to enable history and live view audio to be on by default?
